FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/Makefile

```diff
@@ -0,0 +1,222 @@
+SOURCE_FILES += ${MBED_TLS}/library/nist_kw.c
+SOURCE_FILES += ${MBED_TLS}/library/oid.c 
+SOURCE_FILES += ${MBED_TLS}/library/padlock.c
+SOURCE_FILES += ${MBED_TLS}/library/pem.c 
+SOURCE_FILES += ${MBED_TLS}/library/pk_wrap.c
+SOURCE_FILES += ${MBED_TLS}/library/pk.c 
+SOURCE_FILES += ${MBED_TLS}/library/pkcs5.c 
+SOURCE_FILES += ${MBED_TLS}/library/pkcs11.c 
+SOURCE_FILES += ${MBED_TLS}/library/pkcs12.c 
+SOURCE_FILES += ${MBED_TLS}/library/pkparse.c 
+SOURCE_FILES += ${MBED_TLS}/library/pkwrite.c 
+SOURCE_FILES += ${MBED_TLS}/library/platform_util.c 
+SOURCE_FILES += ${MBED_TLS}/library/platform.c 
+SOURCE_FILES += ${MBED_TLS}/library/poly1305.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_aead.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_cipher.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_client.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_driver_wrappers.c
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_ecp.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_hash.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_mac.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_rsa.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_se.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto_storage.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_crypto.c 
+SOURCE_FILES += ${MBED_TLS}/library/psa_its_file.c 
+SOURCE_FILES += ${MBED_TLS}/library/ripemd160.c 
+SOURCE_FILES += ${MBED_TLS}/library/rsa_internal.c 
+SOURCE_FILES += ${MBED_TLS}/library/rsa.c 
+SOURCE_FILES += ${MBED_TLS}/library/sha1.c 
+SOURCE_FILES += ${MBED_TLS}/library/sha256.c 
+SOURCE_FILES += ${MBED_TLS}/library/sha512.c
+SOURCE_FILES += ${MBED_TLS}/library/ssl_cache.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_ciphersuites.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_cli.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_cookie.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_tls.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_msg.c
+SOURCE_FILES += ${MBED_TLS}/library/ssl_srv.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_ticket.c 
+SOURCE_FILES += ${MBED_TLS}/library/ssl_tls13_keys.c
+SOURCE_FILES += ${MBED_TLS}/library/timing.c 
+SOURCE_FILES += ${MBED_TLS}/library/version_features.c 
+SOURCE_FILES += ${MBED_TLS}/library/version.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509_create.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509_crl.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509_crt.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509_csr.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509write_crt.c 
+SOURCE_FILES += ${MBED_TLS}/library/x509write_csr.c 
+SOURCE_FILES += ${MBED_TLS}/library/xtea.c 
+
+
+# coreMQTT sources 
+SOURCE_FILES += $(COREMQTT)/source/core_mqtt_serializer.c
+SOURCE_FILES += $(COREMQTT)/source/core_mqtt_state.c
+SOURCE_FILES += $(COREMQTT)/source/core_mqtt.c 
+SOURCE_FILES += $(UTILITIES)/backoff_algorithm/source/backoff_algorithm.c
+
+
+# networking specific cflags
+CFLAGS := -DmainCREATE_NETWORKING_DEMO_ONLY=1
```

Comment:
Are these compile time definitions necessary?

Reply:
I was keeping with how the demo is set up for the FreeRTOS+TCP qemu mps2 demo which had these compile definitions, but this could be changed